recipes/libjaylink/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+pushd "${SRC_DIR}" || exit 1
+  meson setup build-${PKG_NAME} \
```

Comment:
I think you can use `$MESON_ARGS`, and then it should install into `lib/...` on linux (and not `lib64`).